remodel/connection.py

```diff
@@ -11,16 +11,26 @@
 
 
 class Connection(object):
-    def __init__(self, db='test', host='localhost', port=28015, auth_key=''):
+    def __init__(self, db='test', host='localhost', port=28015, auth_key='',
+                 user='admin', password=''):
         self.db = db
         self.host = host
         self.port = port
         self.auth_key = auth_key
+        self.user = user
+        self.password = password
```

Comment:
Again my biases, I personally prefer not relying on truthiness of non-boolean types. But I definitely see that in this case it is very unlikely that the extra strictness does not give anything for the price of readability.